nav2_rviz_plugins/src/navigation_dialog.cpp

```diff
@@ -45,8 +45,9 @@ NavigationDialog::NavigationDialog(QWidget * parent)
   auto options = rclcpp::NodeOptions().arguments(
     {"__node:=navigation_dialog_action_client"});
   client_node_ = std::make_shared<rclcpp::Node>("_", options);
-  action_client_ = rclcpp_action::create_client<nav2_msgs::action::NavigateToPose>(client_node_,
-      "NavigateToPose");
+
+  action_client_ = rclcpp_action::create_client<nav2_msgs::action::NavigateToPose>(client_node_, "NavigateToPose");
```

Comment:
Looks like the only change on this line was removing the line break.

This will fail linter checks, the max number of characters per line is 100 given current style guide.

You can check if you do:
```
$ ament_cpplint <filename>
```

Reply:
undo the edit. I didn't know about the 100 character rule.